_data/scala-releases.yml

```diff
@@ -0,0 +1,20 @@
+- category: current_version
```

Comment:
Do you use the category somewhere?

Reply:
the category is not used, as far as I can see but this "field" existed in the current `/download/scala2.md` so I included it here